new/app/appStylesheet.ts

```diff
@@ -147,4 +149,35 @@ setStyleBuilders({
             margin: 0
         }
     },
+    baseTextInput: (config) => {
+        return {
+            backgroundColor: config.themeColorPalette.secondaryBackground,
+            color: config.themeColorPalette.primaryTextColor,
+            borderColor: 'white',     // Set border color to white
+            borderWidth: 1,           // Add a border
+            borderRadius: 5,
+            caretColor: 'black'
```

Comment:
don't create color here. all color should be created in COLORS

Reply:
fixed